src/components/Maps/ChoroplethMap.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from 'react';
+import { 
+    ComposableMap,
+    Geographies,
+    Sphere,
+    Graticule
+} from 'react-simple-maps';
+
+function ChoroplethMap(props) {
+    const { 
+        geoUrl,
+        data,
+        geoMapFn
+    } = props
```

Comment:
Is there any advantage to using a "props" variable vs destructuring the arguments directly?

```suggestion
function ChoroplethMap({ 
        geoUrl,
        data,
        geoMapFn
    }) {
```

Reply:
@sbussard I think this is more of a stylistic thing and I personally find destructuring props in the argument can get pretty grotesque at some point especially for React but more than happen to make the update.